Makefile

```diff
@@ -70,16 +70,16 @@ else
 ifeq ($(LANG_TYPE),RTL)
 	NORM_MODE =3
 	RECODER =--pass_through_recoder --lang_is_rtl
-	GENERATE_BOX_SCRIPT =generate_wordstr_box.py
+	GENERATE_BOX_SCRIPT =generate_wordstr_box.py --rtl
 else
-	NORM_MODE =2
+	NORM_MODE =1
```

Comment:
This matches `src/training/language-specific.sh` and looks good, so it could also be moved to a separate pull request.